src/HttpKernel/InstallationKernel.php

```diff
@@ -67,7 +67,6 @@ public function canBootRealSystem()
     {
         return file_exists($this->getRootDir().'/config/parameters.yml')
             && file_exists($this->getRootDir().'/../system/config/localconfig.php')
```

Comment:
Should this be removed in regards of https://github.com/contao/core-bundle/pull/652 ?